cmd/thermal-recorder/main.go

```diff
@@ -196,9 +199,9 @@ func frameParser(brand, model string) func([]byte, *cptvframe.Frame) error {
 		return nil
 	}
 	switch model {
-	case "lepton3":
+	case lepton3.Model:
```

Comment:
Combining these two case statements into one would make it clearer that the code is the same.